Libraries/LibWeb/IndexedDB/IDBDatabase.cpp

```diff
@@ -211,12 +213,12 @@ WebIDL::ExceptionOr<GC::Ref<IDBTransaction>> IDBDatabase::transaction(Variant<St
     // 4. If any string in scope is not the name of an object store in the connected database, throw a "NotFoundError" DOMException.
     for (auto const& store_name : scope) {
         if (!database->object_store_with_name(store_name))
-            return WebIDL::NotFoundError::create(realm(), "Object store not found"_string);
+            return WebIDL::NotFoundError::create(realm, "Provided scopes does not name a valid object store"_string);
```

Comment:
Nit: "Provided scope"?

Reply:
Changed to a different message that better conveys the parameters intention 👍